package.json

```diff
@@ -146,41 +146,37 @@
     "mocha": "^6.1.4",
     "nyc": "^14.1.1",
     "sinon": "^8.1.1",
-    "ts-node": "^8.10.2",
     "tslint": "^5.18.0",
     "tslint-config-prettier": "^1.18.0",
     "tslint-no-unused-expression-chai": "^0.1.4",
-    "typescript": "^3.3.1",
+    "typescript": "3.9.5",
     "prettier": "^1.19.1",
     "vsce": "^1.76.1",
     "vscode-test": "^1.3.0",
     "npm-run-all": "^4.1.5",
     "lint-staged": "^9.5.0",
     "husky": "^4.2.5",
+    "source-map-support": "^0.5.12",
     "conventional-recommended-bump": "^6.0.9",
     "conventional-changelog-cli": "^2.0.27",
     "cz-conventional-changelog": "^3.0.2",
     "@commitlint/cli": "^8.2.0",
-    "@commitlint/config-conventional": "^8.2.0",
-    "@istanbuljs/nyc-config-typescript": "1.0.1"
+    "@commitlint/config-conventional": "^8.2.0"
   },
   "nyc": {
     "lines": 90,
     "functions": 85,
     "branches": 85,
     "statements": 90,
-    "extension": [
-      ".ts"
-    ],
     "include": [
-      "src/**"
+      "out/src/**"
```

Comment:
Please check that the coverage works correctly (and fails if you increase it to be too high)

Reply:
Done - it works as expected